frontend/components/product-list/sections/FilterableProductsSection/useFacets.tsx

```diff
@@ -49,7 +49,13 @@ export function useFilteredFacets(productList: ProductList) {
       return usefulFacets;
    }, [facets, infoNames, sortBy]);
 
-   if (productList.type === ProductListType.AllTools) {
+   const isToolsPage =
+      productList.type === ProductListType.AllTools ||
+      productList.type === ProductListType.ToolsCategory
+         ? true
+         : false;
```

Comment:
Pretty sure you should always get booleans from `===` anyhow:
```suggestion
      productList.type === ProductListType.AllTools ||
      productList.type === ProductListType.ToolsCategory;
```

Reply:
Resolved b33fc556fa304945d7ec0c7ea2594acc32e3ff6e